src/js/edu-benefits/1995/components/IntroductionPage.jsx

```diff
@@ -73,6 +74,9 @@ class IntroductionPage extends React.Component {
                 afterText="»"/>
           </div>
         </div>
+        <div className="omb-info--container">
+          <OMBInfo resBurden={20} ombNumber="2900-0074" expDate="05-31-2018"/>
```

Comment:
That's copied from the paper form, but I think using our usual vets.gov style is better!